### ISSUE TITLE
Kythe release: update RELEASES.md log [phab:D2090]

### ISSUE THREAD

**Issue by kythe** [google/kythe#2547]
_Created by **schroederc** at 2017-12-08 20:54:34_:

**Comment by kythe** [comment]
_**fromberger** accepted this revision at 2017-12-08 21:03:11_

**Comment by kythe** [comment]
_**zarko** commented inline (context lost during migration) at 2017-12-08 21:30:26_:

```
   - better support for C++{11,14,17}
   - CUDA support
   - improvements to documentation indexing and rendering
   - added a plugin for indexing proto fields in certain string literals
   - type ranges for builtin structural types are no longer destructured (`T*` is now `[T]*`, not `[[T]*]`)
   - decoration of builtin types can be controlled with --emit_anchors_on_builtins
   - namespaces are never defined, only `ref`d
   - old-style `name` nodes removed
   - the extractor now captures more build state, enabling support for `__has_include`
   - anchors involved in `completes` edges now contain their targets mixed with their signatures, making each completion relationship unique
   - support for indexing uses of `make_unique` et al as direct references to the relevant constructor
   - template instantiations can be aliased together with --experimental_alias_template_instantiations, which significantly decreases output size at the expense of lower fidelity
```

**Comment by kythe** [comment]
_**zarko** accepted this revision at 2017-12-08 21:30:26_

**Comment by kythe** [comment]
_**schroederc** wrote at 2017-12-08 21:52:52_:

Add more C++ analysis changes

**Comment by kythe** [comment]
_**schroederc** wrote at 2017-12-08 21:54:03_:

Thanks, zarko!

**Comment by kythe** [comment]
_**schroederc** wrote at 2017-12-08 21:54:54_:

Fix typo

**Comment by kythe** [comment]
_**schroederc** closed via commit ff939d9f90ed6f567c4d92192c80e352c6c378a5 at 2017-12-08 21:57:32_
